src/demo/index.jsx

```diff
@@ -18,13 +18,15 @@ $(document).ready(() => {
 
   // Temporarily use URL parameter to set some state.
   const smallWordSet = window.location.href.indexOf('words=small') !== -1;
+  const loadTrashImages = window.location.href.indexOf('mode=fishvtrash') !== -1;
```

Comment:
the latest `master` has a helper for this:
https://github.com/code-dot-org/ml-activities/blob/41beede2799155ab02fdc61886901665b1060778/src/demo/helpers.js#L73-L76

Reply:
Thanks! I think I used it correctly!